crates/cli-support/src/wit/mod.rs

```diff
@@ -1457,15 +1457,16 @@ fn extract_programs<'a>(
                     "
 
 it looks like the Rust project used to create this wasm file was linked against
-a different version of wasm-bindgen than this binary:
+version of wasm-bindgen that uses a different bindgen format than this binary:
 
   rust wasm file: {}
      this binary: {}
```

Comment:
FWIW This was intended to be a human readable error message where the versions here were human readable and could give an indicator of what to do. With the versions being hashes now, though, this message isn't quite as appropriate as it was before.  Perhaps these indicators could be less emphasized? Or could the version still be serialized somewhere and it's only used for these purposes?

Mostly I'm just thinking that hashes shouldn't be too prominently displayed here.

Reply:
Ah right, I forgot that the versions are now hashes when I was editing this. 

I'll expose the Cargo.toml version in the schema so that be used here and reword the message to more or less suggest that the easiest solution is to make the CLI and wasm-bindgen versions match.